### PR TITLE
feat: Adding Missing User Attribute pags

### PR DESCRIPTION
**Pull request by jgribonvald**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] tests are included
-   [ ] documentation is changed or added
-   [ ] [message properties][] have been updated with new phrases
-   [ ] view conforms with [WCAG 2.0 AA][]

##### Description of change

Providing a pags tester to test if a user has an attribute.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: uPortal-groups/uPortal-groups-pags/src/main/java/org/apereo/portal/groups/pags/testers/MissingAttributeTester.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    /** @return String */
+    public String getAttributeName() {
+        return attributeName;
+    }
+
+    @Override
+    public final boolean test(IPerson person) {
+        Object[] atts = person.getAttributeValues(getAttributeName());
+        if (atts != null && atts.length > 0) {
```
Comment (ChristianMurphy) [inline uPortal-groups/uPortal-groups-pags/src/main/java/org/apereo/portal/groups/pags/testers/MissingAttributeTester.java]: :art: `return atts == null || atts.length < 1`

**Comment by jgribonvald** [comment]
@drewwills so what do you think ?

File: uPortal-groups/uPortal-groups-pags/src/main/java/org/apereo/portal/groups/pags/testers/MissingAttributeTester.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import org.slf4j.LoggerFactory;
+
+/** Class testing a missing attribute on IPerson. */
+public class MissingAttributeTester implements IPersonTester {
+
+    protected final String attributeName;
+
+    protected final Logger logger = LoggerFactory.getLogger(getClass());
+
+    /** @since 5.2 */
```
Comment (drewwills) [inline uPortal-groups/uPortal-groups-pags/src/main/java/org/apereo/portal/groups/pags/testers/MissingAttributeTester.java]: This comment should probably be on the class itself.

**Comment by drewwills** [comment]
Looks clean & correct.

**Pull request merged**